docs/hugo/content/introduction/breaking-changes/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking Changes"
+linkTitle: "Breaking Changes"
+---
+We go to great lengths to avoid breaking changes as much as possible, as we're well aware that they can cause issues for our users. However, they do occasionally occur, so we've committed to providing good documentation each time this occurs.
+
+## Released Breaking Changes
+
+### v2.0.0
+
+Breaking changes are:
+
+* Upgrades from releases prior to v2.0.0-beta.5 are disallowed
+* Alpha CRD versions have been removed
+
+For more information see [v2.0.0 Breaking Changes](./breaking-changes-v2.0.0).
+
+### v2.0.0-beta.4
+
+In the `beta.4` release of Azure Service Operator (ASO) we pivoted to using Azure Swagger API Specifications as the sole source of truth for our code generator. This change brought with it a significant improvement in fidelity - the code we generate is now much closer to what the Azure Swagger API Specifications describe. Unfortunately, this change also brings with it a number of breaking changes requiring simple manual modifications to pre-existing resources.
+
+Breaking changes are:
+
+* The discriminator value for polymorphic types has been changed to match the name of the property used to specify that option.
+* Enumerated properties that previously had a base type but no enumeration values have been updated to include the enumeration values.
+* Objects that were incorrectly generated as nested properties have been directly inlined
+* ARM Id fields are now correctly identified as references, allowing linking to a resource in Kubernetes instead of only in Azure.
+* Status properties that cannot be set by the end user on a Spec that were included in the Spec in error
+* Sub-resources that were incorrectly inlined into the parent resource have been moved to a separate resource.
+* Properties that previously included on Spec but actually had no function have been removed.
+* Validation rules have been tightened, or added to properties that previously had no validation rules.
+* 
+For detailed information, including an exhaustive list of all affected resource properties, see [v2.0.0 Breaking Changes](./breaking-changes-v2.0.0-beta.4).
+
+## Upcoming Breaking Changes
```

Comment:
I feel like this should be at the top, rather than after the released breaking changes?

Or alternatively, it could be in a section under the version it is announced with, as otherwise the "upcoming" list of breaking changes may change over time in a way that doesn't preserve history very well?

Something like this:

```
## v2.0.0

### Upcoming Breaking Changes
...

### Breaking changes
...
```

maybe?

Reply:
Actually I take it back, I think I like it how you have it here, but maybe we should call out the versions it is upcoming in more clearly (and move this to the top I think?)

```
## Upcoming Breaking Changes

### 2.3.0 or later
Beta CRD versions (any version with `v1beta` prefix) will be deprecated no sooner than **v2.3.0**. 
We recommend you start using `v1api` prefixed versions now. 

You can easily swap from a `v1beta` version to a `v1api` version by just replacing `v1beta` with `v1api` in your CRD YAML, the resource shapes are the same.
```

That way we can also have a `2.4.0 or later` section in the future?

Reply:
As suggested, moved to the top with a heading for the version.